pkg/clustermesh/types/types.go

```diff
@@ -4,18 +4,33 @@
 package types
 
 import (
-	"errors"
 	"fmt"
+
+	"github.com/cilium/cilium/pkg/defaults"
 )
 
 const (
 	// ClusterIDMin is the minimum value of the cluster ID
-	ClusterIDMin = 0
-
-	// ClusterIDMax is the maximum value of the cluster ID
-	ClusterIDMax = 255
+	ClusterIDMin    = 0
+	ClusterIDExt511 = 511
```

Comment:
Nit: what does `ClusterIDExt511` represent, exactly?

Reply:
A valid configuration for `--max-connected-clusters`. The flag should accept 255 (defaults.MaxConnectedClusters) or 511